petroly/settings/base.py

```diff
@@ -29,7 +29,7 @@
     'strawberry.django',
     'gqlauth',
     'strawberry_django_jwt.refresh_token',
-    'notifier'
+    'notifier.apps.NotifierConfig'
```

Comment:
I believe `notifier` is suffice , the Config will be discovered automatically

Reply:
I changed it to override `read()` in my early trials to get the bot working under `runserver` and forgot to revert it back